src/components/ChangePlan/ChangePlan.js

```diff
@@ -23,14 +40,27 @@ const ChangePlan = () => {
         <section className="dp-container">
           <div className="dp-rowflex">
             <div className="col-sm-12" style={{ textAlign: 'center' }}>
-              <h1>Compra un plan</h1>
-              <a href={planUrl} class="dp-button button-medium primary-green">
-                Comprar Plan
-              </a>
-              <br />
-              <br />
-              Información del plan Plan Id {planId} - DiscountId - {safeDiscountId} - PromoCode -{' '}
-              {safePromoId}
+              <Card>
+                <div className="dp-content-plans">
+                  <h3>{cardData.name}</h3>
+                  <p>{_('change_plan.description')}</p>
+                  <p>{cardData.descriptionPlan}</p>
+                  <CardPrice currency="US$">{cardData.price}</CardPrice>
+                  <CardAction url={cardData.action.url}>{cardData.action.text}</CardAction>
+                </div>
+              </Card>
+              <Card cssClass="dp-highlighthed">
+                <Ribbon position="top-right">
+                  <span>{_('change_plan.recommended')}</span>
+                </Ribbon>
```

Comment:
Está bien aceptar HTML? por ejemplo podríamos hacer algo así?

```html
                <Ribbon position="top-right">
                  <span>Este es un <strong>Ofertoooon<strong></span>
                </Ribbon>
```

Eso está bueno, si es así, que opinás de quitar el _spam_?

```suggestion
                <Ribbon position="top-right">
                  {_('change_plan.recommended')}
                </Ribbon>
```

Si no es así, si solo vamos a permitir texto plano, que opinás de hacerlo como una propiedad?

```suggestion
                <Ribbon position="top-right" content={_('change_plan.recommended')}></Ribbon>
```

Reply:
Fui mas por la ultima solución ya que en principio seria texto plano, en el caso de que creciera esto se podría cambiar